src/Controller/PlanningController.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Domain\PlanningUpdateDomain;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\JsonResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/planning")
+ */
+class PlanningController extends AbstractController
+{
+    /**
+     * @Route("/update/{action}", name="planning_update", methods={"POST"})
+     */
+    public function update(Request $request, EntityManagerInterface $entityManager, string $action): Response
+    {
+        $json = json_decode($request->getContent(), true);
```

Comment:
php.net/manual/fr/function.json-decode.php => JSON_THROW_ON_ERROR would be easier

Reply:
Discussed : there is no value for this as it would end up by adding a try catch to convert the exception into a BadRequestException